pytraj/all_actions.py

```diff
@@ -169,61 +229,62 @@ def distance(traj=None,
     # TODO: add image, noe, ...
     """compute distance between two maskes
 
-    Parameters
-    ----------
-    traj : Trajectory-like, list of Trajectory, list of Frames
-    mask : str or a list of string or a 2D array-like of integers.
-           If `mask` is a 2D-array, the `image` option is always `False`.
-           In this case, make sure to `autoimage` your trajectory before
-           calling `distance`.
-    frame_indices : array-like, optional, default None
-    dtype : return type, default 'ndarray'
-    top : Topology, optional
-    image : bool, default False
-    n_frames : int, optional, default None
-        only need to provide n_frames if ``traj`` does not have this info
```

Comment:
@sastrys1  I think you mess up with other docs.

Reply:
and so on for other places.

Reply:
My apologies, used yapf but it must have messed stuff up for the file. I fixed the formatting issue.